docs/docs/aztec_connect_sunset.mdx

```diff
@@ -33,61 +28,3 @@ Follow the links for more information about each package.
 - [Block Explorer](https://github.com/AztecProtocol/aztec-connect-explorer)
 - [Alpha SDK](https://github.com/AztecProtocol/aztec-connect/tree/master/yarn-project/alpha-sdk)
 - [Wallet UI](https://github.com/AztecProtocol/wallet-ui)
-
-## Zk.money
```

Comment:
I think we should keep these instructions as they are still relevant for people running the ejector themselves.